src/pykeen/experiments/inverse_stability.py

```diff
@@ -67,7 +67,8 @@ def run_inverse_stability_workflow(
             create_inverse_triples=True,
         ),
     )
-    dataset_name = dataset_instance.get_normalized_name()
+    # FIXME
```

Comment:
revert this back to using get_normalized_name

Reply:
https://github.com/pykeen/pykeen/commit/d073e4a4aea4395aedf8cca9d4589b411fee274f